eng/ci/templates/steps/build-extension-project.yml

```diff
@@ -16,7 +16,7 @@ steps:
   - template: ci/sign-files.yml@eng
     parameters:
       displayName: Sign assemblies
-      folderPath: sdk
+      folderPath: extensions/${{ parameters.ExtensionProjectName }}/src/bin/
       pattern: Microsoft.Azure.Functions.Worker.Extensions*.dll
```

Comment:
Is there a way to block the pipeline from publishing unsigned packages? May be in this pipeline or somewhere else in the flow. Or provide a clear error or at the minimum we can name the package with suffix `-unsigned`. What will be the best practice for this?

Reply:
1ES PT has an opt-in code sign validation step: https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/1es-pipeline-templates/features/sdlanalysis/codesignvalidation

We can evaluate enabling that later